Authenticator/Source/TextFieldRowModel.swift

```diff
@@ -25,20 +25,20 @@
 
 import UIKit
 
-protocol TextFieldRowModel {
-    var label: String { get }
-    var placeholder: String { get }
+struct TextFieldRowModel<Action> {
+    let label: String
+    let placeholder: String
 
-    var autocapitalizationType: UITextAutocapitalizationType { get }
-    var autocorrectionType: UITextAutocorrectionType { get }
-    var keyboardType: UIKeyboardType { get }
-    var returnKeyType: UIReturnKeyType { get }
+    let autocapitalizationType: UITextAutocapitalizationType
+    let autocorrectionType: UITextAutocorrectionType
+    let keyboardType: UIKeyboardType
+    let returnKeyType: UIReturnKeyType
 
-    var value: String { get }
-    var changeAction: (String) -> Form.Action { get }
+    let value: String
+    let changeAction: (String) -> Action
 }
 
-struct IssuerRowModel: TextFieldRowModel {
+func IssuerRowModel(value value: String, changeAction: (String) -> Form.Action) -> TextFieldRowModel<Form.Action> {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 115 characters (line_length)